[esx]/esx_menu_default/client/main.lua

```diff
@@ -12,6 +13,7 @@ local function openMenu(namespace, name, data)
 end
 
 local function closeMenu(namespace, name)
+    CurrentNameSpace = namespace
```

Comment:
Do you want to set this to **nil** or is it supposed to have the previous namespace?

Reply:
I supposed to have the previous namespace.
That's on validate for menu submit.
but now can be **nil**